### ISSUE TITLE
Sending Form Data Tutorial, adding reference for PHP Server app

### ISSUE THREAD

**Issue by RisingThunder** [mdn/sprints#2660]
# Request type

- [ ] Please close this issue, I accidentally submitted it without adding any details
- [ ] New documentation
- [x] Correction or update

# Details

Reference the MDN web docs article <a href="https://developer.mozilla.org/en-US/docs/Learn/Forms/Sending_and_retrieving_form_data">Sending form data</a>, in the sub section "Example: Raw PHP", in the yellow note box.
Could a reference to <a href="http://www.wampserver.com/en/">Wampserver</a> be added to the list of PHP servers, for sake of simplicity for beginners such as myself? Of the two PHP server apps that are listed, AMPPS is "a little much" and MAMP "free version" would not work without installing the time limited full version. After much research, trial and error I found the php server app <a href="http://www.wampserver.com/en/">Wampsever</a> and this video "<a href="https://www.youtube.com/watch?v=D_Fhu_6RuMw&t=">Installing, Troubleshooting and Configuring WAMP server for Windows 10</a>" which was very helpful in quickly setting up this php server for a beginner. This could help other students, save time, and keep moving through this MDN web docs tutorial in a timely manor, 
Thanks!

**Comment by chrisdavidmills** [comment]
Hi there!

Thanks for the suggestion. The problem with this is that WAMP is Windows-only, whereas the options we suggest provide coverage across all platforms. And, while you need to install MAMP and MAMP Pro together, there is no requirement to pay for anything, and MAMP is simple to use.

**Comment by RisingThunder** [comment]
Thanks Chris, that makes sense. 
After a little more researching, I did find if the MAMP PRO is not installed, or if the MAMP PRO demo time trial has expired (as it has for me before I got to play around with it by now), to get the "free" MAMP working with it's contained Apache Server, go to the MAMPS > Preferences > PHP > and set "Standard Version:" to "7.2.14". That simple, but if you don't know, you don't know. Maybe a short comment could be added to help student's following me? Really love these MDN web docs tutorials, think they are the best!!
